MatiasAnas/node-course
# feat: added section 6 practice

## Summary

* Added Section 6 Practice.
* Decided to use ejs library.

## Test Cases

* npm start
* Use the navigation bar to go to the Home route ('/') or to the Users route ('/users').
* Submit the create user form in Home, it should redirect to Users and add the user to the "database".
* Enter an invalid route url to get the Not Found screen.

## Screenshots

### Home:

![Captura de pantalla de 2020-12-28 13-39-38](https://user-images.githubusercontent.com/12876677/103230656-bcc2f900-4914-11eb-88da-1fcef2097b7a.png)

### Users (No users created yet):
![Captura de pantalla de 2020-12-28 13-40-27](https://user-images.githubusercontent.com/12876677/103230662-bf255300-4914-11eb-99ca-ccb041a8d0e9.png)

### Users (Some users created):

![Captura de pantalla de 2020-12-28 13-41-53](https://user-images.githubusercontent.com/12876677/103230668-c0ef1680-4914-11eb-9614-92728da8555c.png)

### Not-Found

![Captura de pantalla de 2020-12-28 13-42-07](https://user-images.githubusercontent.com/12876677/103230670-c2b8da00-4914-11eb-8ff2-41fcd5add47b.png)
